docs/pages/workflow/glossary-of-terms.md

```diff
@@ -33,60 +129,177 @@ The term "eject" was popularized by [create-react-app](https://github.com/facebo
 
-### Expo client
+### Expo Install
 
-The former name for the [Expo Go](#expo-go) app.
+Refers to the command `npx expo install` from [Expo CLI](#expo-cli). This command is used to install NPM packages containing [native modules](#native-module) that work with the currently installed version of `expo` in the project. Not all packages are supported. This command wraps the globally installed [package managers](#package-manager).
+
+### Expo Module Config
+
+A file named `expo-module.config.json` that lives in the root directory of a [native module](#native-module). [Learn more](/modules/module-config/).
 
 ### Expo SDK
 
 The Expo SDK provides access to device/system functionality such as camera, push notification, contacts, file system, and more. Scroll to the SDK API reference in the documentation navigation to see a full list of APIs and to explore them. [Read more about the Expo SDK](/versions/latest/). [Find it on GitHub](https://github.com/expo/expo-sdk).
 
+### Expo Start
+
+Refers to the command `npx expo start` from [Expo CLI](#expo-cli). This command is used to start a local [development server](#development-server) that a [client](#expo-client) connects to in order to interact with the [Metro bundler](#metro-bundler).
+
 ### ExpoKit
 
+> Deprecated technology
+
 ExpoKit is an Objective-C and Java library that allows you to use the [Expo SDK](#expo-sdk) and platform and your existing Expo project as part of a larger standard native project — one that you would normally create using Xcode, Android Studio, or `react-native init`. [Read more](../expokit/eject.md).
 
 **Support for ExpoKit ended after SDK 38. Expo modules can implement support for custom native configuration, and projects that need even more custom native code can [expose their Android Studio and Xcode projects with `expo prebuild`](/workflow/customizing/).**
 
+### Fabric
+
+The React Native rendering system, used to create an manage native views. [Learn more](https://reactnative.dev/architecture/fabric-renderer). This package supports [iOS](#ios) and [Android](#android).
+
+### Flipper
+
+A mobile app debugger. [Learn more](https://fbflipper.com/). We do not enable Flipper in the default [Prebuild](#prebuild) template due to instability concerns.
+
+### FYI
+
+Sometimes referred to as "Expo FYI", this is a collection of tailored solutions to complex issues that lives at [expo.fyi](https://expo.fyi/). FYI links are utilized throughout Expo's developer tooling to help provide a better developer experience to users.
+
+### Global Expo CLI
+
+The package `expo-cli` which was installed globally on the user's machine and used across all projects. This CLI was introduced in SDK 30 (2018), and deprecated in favor of the [Local Expo CLI](#local-expo-cli) in SDK 46 (2022).
+
+### Gradle
+
+The Android package manager and build tools.
```

Comment:
```suggestion
The Android package manager and build tools.
```

Not sure if Gradle is a package manager, but it's a build system for sure. 😉 

Maybe we can reuse Wikipedia page intro here, just with mention of Android somewhere?

> Gradle is a build automation tool for multi-language software development. It controls the development process in the tasks of compilation and packaging to testing, deployment, and publishing.